core/src/jsMain/kotlin/external/BluetoothRemoteGATTDescriptor.kt

```diff
@@ -0,0 +1,15 @@
+package com.juul.kable.external
+
+import com.juul.kable.UUID
+import org.khronos.webgl.BufferDataSource
```

Comment:
Unused import?

Reply:
Apparently I need to get Kotlinter setup. 😄